resources/linux/rpm/code.spec.template

```diff
@@ -1,23 +1,23 @@
 Name:     @@NAME@@
 Version:  @@VERSION@@
 Release:  @@RELEASE@@.el7
-Summary:  Code editing. Redefined.
+Summary:  Azure Data connectivity.
 Group:    Development/Tools
 Vendor:   Microsoft Corporation
-Packager: Visual Studio Code Team <vscode-linux@microsoft.com>
+Packager: Azure Data Studio Team
 License:  @@LICENSE@@
-URL:      https://code.visualstudio.com/
+URL:      https://aka.ms/azuredatastudio
 Icon:     @@NAME@@.xpm
 Requires: @@DEPENDENCIES@@
 AutoReq:  0
 
 %global __provides_exclude_from ^%{_datadir}/%{name}/.*\\.so.*$
 
 %description
-Visual Studio Code is a new choice of tool that combines the simplicity of a code editor with what developers need for the core edit-build-debug cycle. See https://code.visualstudio.com/docs/setup/linux for installation instructions and FAQ.
+Azure Data Studio is a lightweight, cross-platform data management and development tool with connectivity to popular cloud and on-premises databases. For more information, visit https://aka.ms/azuredatastudio.
 
 # Don't generate build_id links to prevent conflicts when installing multiple
-# versions of VS Code alongside each other (e.g. `code` and `code-insiders`)
+# versions of ADS alongside each other (e.g. `azuredatastudio` and `azuredatastudio-insiders`)
```

Comment:
I would use Azure Data Studio instead of ADS here

Reply:
This is just a comment, so ADS should be fine IMO (although it wouldn't hurt to spell it out either)

Reply:
I'll let it be as it's a comment.